docs/manual/scala/releases/Migration16.md

```diff
@@ -22,6 +22,19 @@ We also recommend upgrading to sbt 1.2.8 or later, by updating the `sbt.version`
 
 ## Main changes
 
+### Remoting Artery
+
+Lagom 1.6.0 builds on Akka 2.6.0 that uses a new Akka Remote implementation called Artery. Artery is enabled by default in Lagom and replaces the previous Akka Remote protocol (aka. Akka Remote Classic). If you are using Lagom in a clustered setup, you will need to shutdown all nodes before updating, unless you choose to disable Artery.
+
+To use classic remoting instead of Artery, you need to:
+
+1. Set property `akka.remote.artery.enabled` to `false`. Further, any configuration under `akka.remote` that is specific to classic remoting needs to be moved to `akka.remote.classic`. To see which configuration options are specific to classic search for them in: [`akka-remote/reference.conf`](https://github.com/akka/akka/blob/master/akka-remote/src/main/resources/reference.conf)
```

Comment:
don't link to master. Akka's reference.conf is listed in https://doc.akka.io/docs/akka/2.6/general/configuration.html#listing-of-the-reference-configuration if you prefer a doc link